plans/logs.tf

```diff
@@ -0,0 +1,42 @@
+resource "azurerm_resource_group" "logs" {
+    name     = "${var.prefix}logs"
+    location = "${var.location}"
+    tags {
+        env = "${var.prefix}"
+    }
+}
+
+resource "azurerm_storage_account" "logs" {
+    name                = "${var.prefix}logs"
+    resource_group_name = "${azurerm_resource_group.logs.name}"
+    location            = "${var.location}"
+    account_type        = "Standard_GRS"
```

Comment:
Indeed, I think that using 50GB is fine for now but we may want to increase the size